tests/device/hid_helper_test.py

```
@@ -117,83 +115,34 @@ def test_no_interface_returns_false(self):
-        self.assertTrue(ok1)
-        self.assertTrue(ok2)
-        lock.release()
-
-
 class TestSendMultiple(unittest.TestCase):
 
-    def test_acquires_lock_when_none_passed_and_keeps_it(self):
+    def test_writes_and_frees_lock(self):
         helper, device = _helper()
-        ok, result, lock = helper.send_multiple(CMD, None)
+        ok, result = helper.send_multiple(CMD)
```

Ruff (0.15.15)

[warning] Unpacked variable `result` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)